components/spo2/src/spo2_filter.c

```diff
@@ -1,72 +1,76 @@
 #include "spo2_filter.h"
 
-
 /**
  * spo2_ac_lowpass_no_offset() - 5 Hz digital low-pass filter (without offset)
- * @feedback:	previous filter output
- * @adc_value:	current filter input
+ * @feedback:   previous filter output
+ * @adc_value:  current filter input
  *
  * This sigital filter gives a bound at 5 Hz filter. Additionally it removes an
  * offset of 50% of the input voltage. It is used pre-process the red and ird
  * sensor values.
  */
-static int32_t spo2_ac_lowpass_no_offset(int32_t feedback, int32_t adc_value);
+static int32_t
+spo2_ac_lowpass_no_offset(int32_t feedback,
+                          int32_t adc_value);
+
 
 /**
  * spo2_dc_lowpass() - 0.5 Hz digital low-pass filter
- * @feedback:	previous filter output
- * @adc_value:	current filter input
+ * @feedback:   previous filter output
+ * @adc_value:  current filter input
  *
  * This sigital filter gives a bound at 0.5 Hz filter. It is used pre-process
  * the red and ird sensor values.
  */
-static int32_t spo2_dc_lowpass(int32_t feedback, int32_t adc_value);
+static int32_t
+spo2_dc_lowpass(int32_t feedback,
+                int32_t adc_value);
 
 
 void
-spo2_filter
-(spo2_input_sample_t *sample, spo2_adc_sample_t *raw_data)
+spo2_filter(spo2_input_sample_t *sample,
+            spo2_adc_sample_t   *raw_data)
 {
-	/*
-	 * 5Hz low-pass filtering and removing the 2.5 V offset 
-	 * from the amplified AC input values
-	 */
-	sample->ird_ac = spo2_ac_lowpass_no_offset(sample->ird_ac,
-						   (uint16_t)raw_data->ird_ac);
-	sample->red_ac = spo2_ac_lowpass_no_offset(sample->red_ac,
-			                           (uint16_t)raw_data->red_ac);
+    /*
+     * 5Hz low-pass filtering and removing the 2.5 V offset
+     * from the amplified AC input values
+     */
+    sample->ird_ac = spo2_ac_lowpass_no_offset(sample->ird_ac,
+                                               (uint16_t) raw_data->ird_ac);
```

Comment:
i am not sure about spaces after casts. I can't find anything about it right now. I would prefer no spaces, as in my opinion , it's easier to skip over the cast while reading the code. If the cast is close to the variable, it's more unlikely to miss the cast

Reply:
I'm no fan of inline casts at all, if you need to cast, you should always use a correctly typed variable as intermediate.

Reply:
increases loc, but i think this would be a good alternative, we should think about if we want to enforce it

Reply:
removed the space